### PR TITLE
Add script to build and install OpenOCD with XVC support

### PR DESCRIPTION
**Pull request by augustofg**

### CODE REVIEW

File: bin/sirius-script-install-openocd.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+set -e
+
+cd /root
+rm -rf /root/openocd
+git clone https://github.com/lnls-dig/openocd
+cd openocd
+git checkout xvc-tcp-support
+./bootstrap
+./configure --enable-xvc --prefix=/usr/local
+make -j 16
```
Comment (ericonr) [inline bin/sirius-script-install-openocd.sh:12]:
```suggestion
make -j $(nproc)
```

**Review by ericonr — COMMENTED** [review]
Should there be a comment anywhere to explain that it needs to run as root?

**Comment by augustofg** [comment]
yes

File: bin/sirius-script-install-openocd.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -e
+
+cd /root
+rm -rf /root/openocd
+git clone https://github.com/lnls-dig/openocd
+cd openocd
+git checkout xvc-tcp-support
```
Comment (gustavosr8) [inline bin/sirius-script-install-openocd.sh:9]:
```suggestion
git clone https://github.com/lnls-dig/openocd --branch xvc-tcp-support --single-branch
cd openocd
```

**Comment by gustavosr8** [comment]
Do we have `libtool` installed on the GOP PCs?

https://github.com/lnls-dig/openocd/blob/938b4992e76a979e8bbee123864edda5af22607a/bootstrap#L9-L16

**Comment by augustofg** [comment]
> Do we have `libtool` installed on the GOP PCs?
> 
> https://github.com/lnls-dig/openocd/blob/938b4992e76a979e8bbee123864edda5af22607a/bootstrap#L9-L16

Well, it seems to be installed on lnls451-linux at least.

File: bin/sirius-script-install-openocd.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# This should be run as root
+
+set -e
+
+cd /root
+rm -rf /root/openocd
+git clone https://github.com/lnls-dig/openocd --branch xvc-tcp-support --single-branch
```
Comment (ericonr) [inline bin/sirius-script-install-openocd.sh:9]: Nit: options after `clone`, before positional argument (repo URL)

**Review by ericonr — APPROVED** [review]

**Review by gustavosr8 — APPROVED** [review]

**Review by fernandohds564 — APPROVED** [review]

**Pull request merged**